benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -678,5 +682,8 @@ msgstr ""
 "The page you are looking for might be somewhere else or may not exist "
 "anymore."
 
+#, fuzzy
+#| msgid "core.icons.sadbus"
+msgctxt "image alt text"
 msgid "core.icons.sadbus"
```

Comment:
I don't know why this fuzzy is here!

Reply:
It can happen when `gettext` thinks the string has changed. 

These should be manually reviewed and never committed -- when we run `python manage.py compilemessages` (in the init script) it skips all `fuzzy` entries so they never make it into the compiled language files.

https://docs.djangoproject.com/en/4.2/topics/i18n/translation/

> **Fuzzy entries**
>
> [makemessages](https://docs.djangoproject.com/en/4.2/ref/django-admin/#django-admin-makemessages) sometimes generates translation entries marked as fuzzy, e.g. when translations are inferred from previously translated strings. By default, fuzzy entries are not processed by [compilemessages](https://docs.djangoproject.com/en/4.2/ref/django-admin/#django-admin-compilemessages).